src/main/config/feature.h

```diff
@@ -53,7 +53,7 @@ typedef enum {
     FEATURE_SOFTSPI = 1 << 26,
     FEATURE_ESC_SENSOR = 1 << 27,
     FEATURE_ANTI_GRAVITY = 1 << 28,
-    FEATURE_DYNAMIC_FILTER = 1 << 29,
```

Comment:
(unrelated change ... also trailing comma is cleaner in git diff

Reply:
Done.